src/test/java/spoon/test/fieldaccesses/FieldAccessTest.java

```diff
@@ -415,7 +416,7 @@ class CounterScanner extends CtScanner {
 			@Override
 			public <T> void visitCtFieldWrite(CtFieldWrite<T> fieldWrite) {
 				visited++;
-				assertEquals("a", ((CtVariableWrite) fieldWrite.getTarget()).getVariable().getSimpleName());
+				assertEquals("a", ((CtVariableRead<?>) fieldWrite.getTarget()).getVariable().getSimpleName());
```

Comment:
This test was failing as I said [here](https://github.com/INRIA/spoon/pull/4836#issuecomment-1212940313). I think it is supposed to be `CtVariableRead` because `fieldWrite.getTarget` evaluates to `a` in `a.l` in this [file](https://github.com/INRIA/spoon/blob/ae305c31620d3a3fdd704119babe006636779593/src/test/resources/noclasspath/FieldAccessRes.java#L5).

Reply:
Probably I had some local changes and that is why it was failing. I have reverted them in the latest [commit](https://github.com/INRIA/spoon/pull/4836/commits/59ccc5a6c5e454197ae795b72cfcbcdfa874ed8e).